sourcecode/apis/contentauthor/resources/assets/react/components/RichEditor/RichEditor.js

```diff
@@ -110,7 +112,11 @@ class RichEditor extends React.Component {
                 if (this.props.name) {
                     config.title = this.props.name;
                 }
-                config.language = this.props.intl.locale.toLocaleLowerCase();
+                if (this.props.language && this.props.language !== '') {
```

Comment:
The `!== ''` check is redundant, as if the first check succeeds, the second will never fail.